web/cache.go

```diff
@@ -18,15 +18,10 @@ package web
 import (
 	weakrand "math/rand"
 	"sync"
-	"time"
 )
 
 var cacheSize = 100
 
-func init() {
-	weakrand.Seed(time.Now().UnixNano())
```

Comment:
is this not needed anymore?

Reply:
No, deprecated and not needed since Go 1.20.